### PR TITLE
Use Infra's abort when

### PR DESCRIPTION
**Pull request by annevk**
See https://github.com/whatwg/infra/pull/172.

***
<a href="https://whatpr.org/fetch/702.html" title="Last updated on Apr 26, 2018, 12:51 PM GMT (fe99602)">Preview</a> | <a href="https://whatpr.org/fetch/702/0dec453...fe99602.html" title="Last updated on Apr 26, 2018, 12:51 PM GMT (fe99602)">Diff</a>

### CODE REVIEW

File: fetch.bs [hunk 1]
```diff
@@ -3310,7 +3315,7 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
  <li><p>Let <var>httpRequest</var> be null.
 
  <li>
-  <p>Run these steps, but abort if the ongoing fetch is <a for=fetch>terminated</a>:</p>
+  <p>Run these steps, but <a>abort when</a> the ongoing fetch is <a for=fetch>terminated</a>:</p>
```
Comment (jakearchibald) [inline fetch.bs]: My bad in the original PR, but the `</p>` is inconsistent.

File: fetch.bs [hunk 2]
```diff
@@ -1566,7 +1566,7 @@ for each associated <a for="fetch group">fetch record</a> whose
  <li><p>Let <var>connection</var> be null.
 
  <li>
-  <p>Run these steps, but abort if the ongoing fetch is <a for=fetch>terminated</a>:</p>
+  <p>Run these steps, but <a>abort when</a> the ongoing fetch is <a for=fetch>terminated</a>:</p>
```
Comment (jakearchibald) [inline fetch.bs]: Remove closing `</p>`

File: fetch.bs [hunk 3]
```diff
@@ -2403,7 +2403,7 @@ the request.
 
 <ol>
  <li>
-  <p>Run these steps, but abort if the ongoing fetch is <a for=fetch>terminated</a>:</p>
+  <p>Run these steps, but <a>abort when</a> the ongoing fetch is <a for=fetch>terminated</a>:</p>
```
Comment (jakearchibald) [inline fetch.bs]: Remove closing `</p>`

File: fetch.bs [hunk 4]
```diff
@@ -2555,7 +2555,7 @@ with a <i>CORS flag</i> and <i>recursive flag</i>, run these steps:
  <li><p>Let <var>response</var> be null.
 
  <li>
-  <p>Run these steps, but abort if the ongoing fetch is <a for=fetch>terminated</a>:</p>
+  <p>Run these steps, but <a>abort when</a> the ongoing fetch is <a for=fetch>terminated</a>:</p>
```
Comment (jakearchibald) [inline fetch.bs]: Remove closing `</p>`

File: fetch.bs [hunk 5]
```diff
@@ -3768,7 +3773,7 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
 
 
  <li>
-  <p>Run these steps, but abort if the ongoing fetch is <a for=fetch>terminated</a>:</p>
+  <p>Run these steps, but <a>abort when</a> the ongoing fetch is <a for=fetch>terminated</a>:</p>
```
Comment (jakearchibald) [inline fetch.bs]: Remove closing `</p>`

File: fetch.bs [hunk 6]
```diff
@@ -3865,7 +3870,7 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
   <p class="note no-backref">This construction operation will not throw an exception.
 
  <li>
-  <p>Run these steps, but abort if the ongoing fetch is <a for=fetch>terminated</a>:</p>
+  <p>Run these steps, but <a>abort when</a> the ongoing fetch is <a for=fetch>terminated</a>:</p>
```
Comment (jakearchibald) [inline fetch.bs]: Remove closing `</p>`

File: fetch.bs [hunk 7]
```diff
@@ -3941,7 +3946,7 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
 
   <ol>
    <li>
-    <p>While true, breaking if the ongoing fetch <a for=fetch lt=terminated>terminates</a>:
+    <p>While true, <a>abort when</a> the ongoing fetch <a for=fetch lt=terminated>terminates</a>:
```
Comment (jakearchibald) [inline fetch.bs]: This is the only one I'm unsure of, as it feels like the "abort" clause is inside the while clause, whereas it's really the other way around.

Comment (annevk) [reply fetch.bs]: That's a good point, I guess I should embrace the additional nested step you suggested.

**Comment by jakearchibald** [comment]
LGTM!

**Pull request merged**